src/assets/scripts/client/aircraft/StripView/StripViewModel.js

```diff
@@ -448,7 +449,7 @@ export default class StripViewModel extends BaseModel {
         this.$aircraftTypeView.text(this._aircraftType);
         this.$cidView.text(this.cid);
         this.$transponderView.text(this._transponder);
-        this.$assignedAltitudeView.text(this._assignedAltitude);
+        this.$assignedAltitudeView.text(_round(this._assignedAltitude));
```

Comment:
this is a good attempt, however, the `_redraw()` method should not contain any logic or manipulations.  The use of `_round()` should instead occur within both the `_init()` and `_updateStripView()` methods.

Reply:
and, since you're in this method, could you also add that note to the docblock and update the `method` name?